Verse/src/Schemas/JSON/Reader.cs

```diff
@@ -212,7 +212,7 @@ public void Stop(ReaderState state)
 			};
 		}
 
-		private BrowserMove<TElement> ReadToArrayFromObject<TElement>(ReaderState state, Func<TElement> constructor,
+		private BrowserMove<TElement> ReadToArrayFromValue<TElement>(ReaderState state, Func<TElement> constructor,

```

Comment:
Why this renaming? Seems we're reading an object here?

Reply:
sorry.... fixed!